src/index.html

```diff
@@ -1,13 +1,431 @@
+                <div class="process__arrow"></div>
+                <p class="process__text">Learn more</p>
+              </a>
+            </div>
+          </article>
+        </section>
+
+        <section class="testimonials" id="testimonials">
+          <h2 class="testimonials__title">
+            <p class="testimonials__title-text">What</p>
+            <p class="testimonials__title-text">people say</p>
+          </h2>
+
+
+          <article class="testimonials__card-container">
+            <div class="testimonials__card">
+              <img
+              src="images/Testimonials/Tal.Gilad.png"
+              alt="Tal Gilad"
+              class="testimonials__photo"
+              >
+
+              <p class="testimonials__text">AIR’s ideas are refreshing and out of the box.
+                Authentic team that focuses on the important path of the brand.
+              </p>
+              <p class="testimonials__person">Tal Gilad</p>
+              <p class="testimonials__position">Teach for America</p>
+            </div>
+
+            <div class="testimonials__card">
+              <img
+              src="images/Testimonials/Azadeh.Hawkins.png"
+              alt="Tal Gilad"
+              class="testimonials__photo"
+              >
+
+              <p class="testimonials__text">AIR is an exceptional agency that leads with creative talent,
+                first-class account servicing.
+              </p>
+              <p class="testimonials__person">Azadeh Hawkins</p>
+              <p class="testimonials__position">Hawkins Consulting</p>
+            </div>
+
+            <div class="testimonials__card">
+              <img
+              src="images/Testimonials/Michel.Grover.png"
+              alt="Tal Gilad"
+              class="testimonials__photo"
+              >
+
+              <p class="testimonials__text">AIR raises the agency bar to stratospheric
+                heights on both creative output and client service.
+              </p>
+              <p class="testimonials__person">Michel Grover</p>
+              <p class="testimonials__position">Hulu</p>
+            </div>
+          </article>
+        </section>
+
+        <section class="vision">
+          <article class="vision__container">
+            <h2 class="vision__title">Vision, Passion, Results</h2>
+            <p class="vision__paragraph">We are sure that first-rate job is possible only if all three components are united.</p>
+            <a href="#send-us-a-message" class="vision__button-container">
+              <button class="vision__button">Apply</button>
+            </a>
+          </article>
+        </section>
+
+        <div class="page__bottom">
+          <section class="send-us-a-message" id="send-us-a-message">
+            <article class="send-us-a-message__container">
+              <h2 class="send-us-a-message__title">Send us  a message</h2>
+
+              <form action="#"
```

Comment:
```suggestion
              <form action="#"
                 name="sent-us"
                 class="send-us-a-message__form"
              >
```

please fix indentation